qbft/state.go

```diff
@@ -31,7 +31,7 @@ type IConfig interface {
 }
 
 type Config struct {
-	Signer         types.SSVShareSigner
+	SSVShareSigner types.SSVShareSigner
```

Comment:
this should probably be just `ShareSigner` like below, and the same for method names

tbh im not sure that we even need the word 'SSV' in the structs, but it's not too bad, lmk what you think @GalRogozinski @MatheusFranco99

Reply:
I agree the word `SSV` here is redundant. I'll remove it, thx